src/components/addTranslation/AddTranslation.tsx

```diff
@@ -15,8 +15,11 @@ const mapState = (state: IState) => ({
 });
 
 const validationSchema = Yup.object().shape({
-  name: Yup.string(),
+  title: Yup.string().required("Title is required"),
```

Comment:
You could also add .min() so that the title isn't eg. just one letter or number.